src/js/calculate.js

```diff
@@ -0,0 +1,67 @@
+import {
+  LENGTH_LIMIT_OPERATOR,
+  MAX_NUMBER_LENGTH,
+  MAX_OPERATOR_LENGTH,
+  OPERATORS,
+  SPLIT_NUMBER_ARRAY_LENGTH,
+} from './constants.js';
+import checkNotNumber from './regex.js';
+
+const handleDecimalPointFloor = (countedNumber) => String(Math.floor(countedNumber));
+
+const handleCalculate = (beforeNumber, operator, afterNumber) => {
+  const { plus, minus, multiplication, divide } = OPERATORS;
+  switch (operator) {
+    case `${plus}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) + parseInt(afterNumber, 10));
+    case `${minus}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) - parseInt(afterNumber, 10));
+    case `${multiplication}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) * parseInt(afterNumber, 10));
+    case `${divide}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) / parseInt(afterNumber, 10));
+    default:
+      break;
+  }
+};
```

Comment:
기왕 함수를 사용해 반복을 줄인다면 값의 형식에 관련된 로직과 실제 비즈니스와 관련된 로직을 분리해서 관리해보는 건 어떨까요?
지금 함수에서는 

<값의 형식>
number <-> string 변환, floor

<비즈니스 로직>
실제 operation

정도가 되겠네요!

아래는 예시일 뿐이니 참고만해주세요!
```suggestion
const operationMap = {
  [OPERATORS.plus]: (left, right) => left + right,
  [OPERATORS.minus]: (left, right) => left - right,
  [OPERATORS.multiplication]: (left, right) => left * right,
  [OPERATORS.plus]: (left, right) => left / right,
}

// 더 좋은 이름이 있을 것 같네요..!
const evaluateStringAsNumber = (cb, ...args) => String(cb(...args.map(Number)))

const formatCalculateResult = Math.floor

const handleCalculate = (beforeNumber, operator, afterNumber) => {
  const operation = operationMap[operator]
  if (!operation) { return null }
  
  return evaluateStringAsNumber(
    formatCalculateResult,
    evaluateStringAsNumber(operation, beforeNumber, afterNumber)
  )
};
```

Reply:
요거는 굳이 반영하지 않으셔도 좋을 것 같아요!